mergify_engine/rules/__init__.py

```diff
@@ -521,6 +522,9 @@ def ChecksTimeout(v: str) -> datetime.timedelta:
                 voluptuous.Required("checks_timeout", default=None): voluptuous.Any(
                     None, voluptuous.All(str, voluptuous.Coerce(ChecksTimeout))
                 ),
+                voluptuous.Required("draft_bot_account", default=None): voluptuous.Any(
+                    None, str
```

Comment:
It can't because the template needs a `context.Context`, we can't pick the draft one as it has not been created yet. And we cannot pick the pull requests that compose the TrainCar because we need only one `context.Context` and we can't random pick one pull request.

Reply:
The template for `bot_account` was useful only to use `{{ author }}`. This doesn't make sense to pick the author of one of the original pull request and this will confuse a lot people to see merge-queue PR opened with names.